sidebars.js

```diff
@@ -255,113 +255,116 @@ module.exports = {
         },
       ],
       Optimize: [
-        optimizeLink("What is Optimize?", "what-is-optimize"),
+        optimizeLink("What is Optimize?", "components/what-is-optimize"),
```

Comment:
And sidebars on the main docs side also get `components/ ` as a prefix for all optimize links.